lib/kitchen/verifier/pester.rb

```diff
@@ -159,7 +175,42 @@ def prepare_command
       #
       # @return [String] a command string
       def run_command
-        really_wrap_shell_code(run_command_script)
+        really_wrap_shell_code(invoke_pester_scriptblock)
+      end
+
+      # Resolves the remote Downloads path from the verifier root path, 
+      # unless they're absolute path (starts with / or C:\)
+      # This updates the config[:downloads], nothing (nil) is returned.
+      #
+      # @return [nil] updates config downloads
+      def resolve_downloads_paths!
+        info("Resolving Downloads path from config.")
+        
+        config[:downloads] = config[:downloads]
+          .map do |source, destination|
+            source = source.to_s
+            info("  resolving remote source's absolute path.")
+            if !source.match?('^/|^[a-zA-Z]:[\\/]') # is Absolute?
+              info("  '#{source}' is a relative path, resolving to: #{File.join(config[:root_path], source)}")
+              source = File.join(config[:root_path], source.to_s).to_s
+            end
```

Comment:
I'd prefer we use standard library functions if we can; [this SO answer](https://stackoverflow.com/questions/1906823/given-a-path-how-to-determine-if-its-absolute-relative-in-ruby) indicates this kind of functionality should be built in.

I'm guessing this is done this way because we can't directly resolve a path on the remote host; does Test Kitchen provide anything we can use here?

Reply:
I couldn't find a better way for a few reasons:
- it's a remote file and I did not find a way to do this (especially at that time in the process).
- If the file does not exist, you can't say if the leaf is a file or a directory (i.e. no extension on Linux)

I spent a lot of time going in circle, that does not mean someone with more experience can't find a better way!

Reply:
If it works for now it's good enough. I'm not experienced enough to say there is for sure a better way here, but maybe someone else will in future. 🙂